mac-os-build/util.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2015 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# File: util.py
+# Purpose:
+#   Various common functions for use in build scripts.
+
+
+import sys
+import os
+from distutils.spawn import find_executable
+
+import defaults
```

Comment:
This and `check_build_config()` is dead code that will never be used in this scenario. If you remove it, you can also remove the `defaults.py` file.